src/scripts/main.js

```diff
@@ -1,29 +1,58 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const validTypes = ['success', 'error', 'warning'];
+
+  if (!validTypes.includes(type)) {
+    throw new Error(
+      `Invalid notification type: "${type}". Expected one of ${validTypes.join(', ')}.`,
+    );
+  }
```

Comment:
This is a good use of error checking to make sure the notification type is valid. However, the error message could be more descriptive. For example, you could include what types are valid.